### ISSUE TITLE
Meteor 0.8.0-rc0 - Blaze - fails building on line 3 in observatory.html

### ISSUE THREAD

**Issue by pwujek** [superstringsoftware/observatory#37]
`While building package 'observatory':
mars/observatory.html:3: Reactive HTML attributes must either have a constant name or consist of a single {{helper}} providing a dictionary of names and values.  A template tag of type BLOCKOPEN is not allowed here.
...="id_logs_bootstrap" {{#if isDynamic}}cla...
                        ^
`

**Comment by devzsolt** [comment]
I can confirm the issue with the final Meteor 0.8.0 too. Please make it compatible with Blaze.

**Comment by dandv** [comment]
This project is [apparently no longer maintained](https://github.com/jhoxray/observatory/issues/34#issuecomment-39054981).

**Comment by devzsolt** [comment]
You're right, thanks for the info. Do you have perhaps an idea how can I substitute Observatory? I like it's features, but Meteor 0.8 is much more important for my project.

**Comment by dandv** [comment]
@miu-zsolt: you can probably patch [observatory.html](https://github.com/jhoxray/observatory/blob/master/mars/observatory.html) after reading a bit through https://github.com/meteor/meteor/wiki/Using-Blaze#html-must-be-well-formed
